src/sonic-bgpcfgd/bgpmon/bgpmon.py

```diff
@@ -153,7 +153,7 @@ def main():
     try:
         bgp_state_get = BgpStateGet()
     except Exception as e:
-        syslog.syslog(syslog.LOG_ERR, "{}: error exit 1, reason {}".format(THIS_MODULE, str(e)))
+        syslog.syslog(syslog.LOG_ERR, "{}: error exit 1, reason {}".format("THIS_MODULE", str(e)))
```

Comment:
How about renaming this to "BgpStateGet" or something more useful than "THIS_MODULE"

Reply:
@gechiang may be you can update that for master. this pr is basically for fixing the issue which is not present in master.

Reply:
@abdosi When this config error kicks in and this change is in place, what do you see in the syslog?  Can you capture that?
Thanks!